src/phonebook/PhoneBook.java

```diff
@@ -0,0 +1,35 @@
+package phonebook;
+
+import java.util.*;
+
+public class PhoneBook {
+
+    private Map<String, List<Note>> phoneBook;
```

Comment:
I think you can get rid of this map. It makes your implementation complex, and we do not want it. Using ArrayList would be good enough.